packages/familie-visittkort/package.json

```diff
@@ -31,12 +31,14 @@
         "prop-types": "^15.8.1"
     },
     "devDependencies": {
+        "@navikt/aksel-icons": "5.x",
         "@navikt/ds-css": "2.x",
         "@navikt/ds-react": "2.x",
         "@navikt/ds-tokens": "2.x",
         "styled-components": "^5.3.5"
     },
     "peerDependencies": {
+        "@navikt/aksel-icons": "5.x",
         "@navikt/ds-css": "2.x || 3.x || 4.x",
```

Comment:
Bør kanskje åpne opp for v5 av ds-bibliotekene også? 
`        "@navikt/ds-css": "2.x || 3.x || 4.x || 5.x",`